### DISCUSSION TITLE
aula iniciando projeto

### DISCUSSION THREAD

**Discussion by WansuViana** [balta-io/2801#38] [Perguntas e Respostas] [answered]
Professor bom dia/tarde/noite.
meu código estar igual ao seu, mas mesmo assim a mensagem "primeiro valor" não aparece quando digito dotnet run.
Continua com hello world, aonde eu errei? desde já obrigado.

![Captura de tela 2022-02-06 145236](https://user-images.githubusercontent.com/98564091/152694250-49a995ee-a699-4abc-b64b-031bca712dec.png)

**Comment by niziul** [comment, marked as answer]
Aparentemente você esqueceu de salvar as modificações. Use as teclas `"Ctrl"` + `"S"` e  veja se funciona. 

Lembre-se! Todas às vezes que você modificar seu arquivo .cs é necessário salvar.

Execução do seu código:
![](https://i.ibb.co/dPjVskQ/imagem-2022-02-06-214852.png)

**Reply by andrebaltieri** [reply]
Muito obrigado pela ajuda @NiziulLuizin 💜

**Comment by WansuViana** [comment]
Mano muito obrigado!!!!!!!!! Travei nisso domingo todo.

**Comment by Dallonso** [comment]
Boa noite @WansuViana 

Aquela bolinha branca na aba do seu programa significa que tem alterações no código que não foram salvas.
![image](https://user-images.githubusercontent.com/15056126/153518658-440a28f5-9641-4e94-8e5c-38fc018c06e0.png)

Uma configuração que habilitei em meu VScode foi o auto save:

![image](https://user-images.githubusercontent.com/15056126/153518368-0cada82a-fc17-4aef-8fa2-b1c6942a0eac.png)

Acredito ser bem útil.

At.te,

**Reply by WansuViana** [reply]
@Dallonso muito obrigado, ativei essa opção ajuda bastante.
